apiary.apib

```diff
@@ -8280,6 +8281,37 @@ Create a ticket.
             + type: `ticket` (string)
             + id: `f29abf48-337d-44b4-aad4-585f5277a456` (string)
 
+### tickets.update [POST /tickets.update]
+
+Update a ticket.
+
++ Request (application/json)
+    + Attributes (object)
+        + id: `f29abf48-337d-44b4-aad4-585f5277a456` (string, required)
+        + subject: `My ticket subject` (string, optional)
+        + description: `My ticket details` (string, optional) - Uses Markdown formatting
+        + ticket_status_id: `46156648-87c6-478d-8aa7-1dc3a00dacab` (string, optional)
```

Comment:
Always use backticks for properties with underscores in them. So `\`ticket_status_id\`\. In this case, because there are two of them, it's not an issue, but an uneven amount of underscore characters messes with the Markdown syntax.

Reply:
And I can't seem to format backticks inside backticks, but I'll assume you know what I mean 😄